docs/Zigbee.md

```diff
@@ -1,4 +1,5 @@
-!!! info "Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi"
+<!-- Required extensions: pymdownx.betterem, pymdownx.tilde, pymdownx.emoji, pymdownx.tasklist, pymdownx.superfences -->
+**Zigbee2Tasmota serves as a gateway for devices connected to a Zigbee wireless network to bridge their communications over to Wi-Fi**
```

Comment:
restore formatting to what it was

Reply:
Now I understand! My editor does not support the !!! info or !!! example... I have to redo it.

Reply:
Removed in latest commit. Restored the formatting, too.